OMCompiler/Compiler/BackEnd/ResolveLoops.mo

```diff
@@ -600,15 +597,19 @@ algorithm
       Integer cn1;
       list<Integer> rest, assigned = {}, unassigned = {};
     case cn1::rest algorithm
-      accCrossNodes:=cn1::accCrossNodes;
+      if not listMember(cn1, accCrossNodes) then
```

Comment:
I saw many uses of `listMember()`, that might be inefficient in the long run depending on how large theses lists can get. Sometimes using `UnorderedMap` with constant lookup speed is better.

Reply:
I tried AvlSetInt with log(N) lookup speed, but that caused a dramatic increase in memory usage.
Currently the speed is sufficient with listMember(). Of course, that could be improved.

Reply:
I guess we were leaking memory, don't know much about `Avl`. I do not even understand why we use all this `Avl` stuff, `UnorderedMap` and `UnorderedSet` is better in almost all regards i think. Especially the member lookup is constant with respect to `n`.

Probably a seperate discussion though, maybe we should have a makover of all these lookup things in general. If `listMember()` is sufficiently fast it is okay for now i think.